### DISCUSSION TITLE
Regarding usage of Vercel to publish an app

### DISCUSSION THREAD

**Discussion by viksof** [vercel/community#3689] [Help] [answered]
### Question

How to publish a fully developed app on Vercel?

**Comment by amyegan** [comment, marked as answer]
Hey @viksof. If you're new to Vercel, I recommend starting with this guide: https://vercel.com/docs/getting-started-with-vercel

You can learn more about [deploying to Vercel](https://vercel.com/docs/deployments/overview#), [supported frameworks](https://vercel.com/docs/frameworks#), and additional features in our [documentation](https://vercel.com/docs) and [guides](https://vercel.com/guides?query=deploy).

**Comment by viksof** [comment]
Hi,

Thank you for your email.

Thanks for the information.

I am about to develop my fully functional web application (pertaining to
any particular industry) from scratch and deploy it on Vercel platform and
retrieve its website address and put it in my resume or Curriculum Vitae
(CV) for sending it to tech companies in Bangalore city in Karnataka state
of the country India for processing for any suitable job openings in their
companies.

May I please know once I have a fully functional web application
(pertaining to any particular industry) ready with me, how can I deploy
that web application on Vercel web hosting platform (without paying any
money from my end) so that the web application gets visibility on the
internet for people (especially tech recruiters in companies) to view it
and shortlist my resume or Curriculum Vitae (CV) for any suitable job
openings related to my profile in their companies?

I am hereby attaching a soft copy of my resume or Curriculum Vitae (CV) in
this email for your reference.(pls note that in my resume or CV, I have
included a gitpod profile website link of mine in which I have done a
project on the creation of a ecommerce web application)

May I please know how can I make use of this profile of mine to collaborate
with other vercel users on software development projects and present my
resume to tech recruiters for consideration for processing my resume for
suitable job openings in their companies?

Can you pls let me know about these things?

Thanks and best regards,

Vikas
